### PR TITLE
Creates a super mutant loadout

### PR DESCRIPTION
**Pull request by Pokebutt**
## Pre-Merge Checklist

- [ ] You tested this on a local server.
- [ ] This code did not runtime during testing.
- [ ] You documented all of your changes.

## Changelog

:cl:
add: Added new things
add: Added more things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

File: code/datums/diseases/f13_fev.dm [hunk 1]
```diff
@@ -84,6 +84,21 @@
...
 				mood.setSanity(SANITY_INSANE) // You're happy, aren't you?
+// FEV - R0CK3T69
+/datum/disease/transformation/mutant/superdebug
+	agent = "FEV-R0CK3T69 toxin strain" // The unstable one.
+	desc = "A megavirus, with a protein sheath reinforced by ionized hydrogen. This variant has been mutated by radiation and will turn the affected person into something less horrifying."
+	new_form = /mob/living/carbon/human/species/smutant
+	possible_forms = list(/mob/living/carbon/human/species/smutant = 1)
+	stage5 = list("<span class='reallybig hypnophrase'>Simple! Efficient! Glorious!</span>")
+
+/datum/disease/transformation/mutantdebug/super/stage_act()
```
Comment (out-of-phaze) [inline code/datums/diseases/f13_fev.dm:95]: This doesn't match the definition above. Should it be `mutant/superdebug` or `mutantdebug/super`?

File: code/datums/diseases/f13_fev.dm [hunk 2]
```diff
@@ -84,6 +84,21 @@
...
+	..()
+
+	switch(stage)
```
Comment (out-of-phaze) [inline code/datums/diseases/f13_fev.dm:98]: This should just be an `if(stage == 2 && prob(8))`.

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 3]
```diff
@@ -872,6 +872,13 @@
 	race = /datum/species/abductor
 	mutationtext = "<span class='danger'>The pain subsides. You feel... alien.</span>"
 
+/datum/reagent/mutationtoxin/smutant
+	name = "Mutie Mutation Toxin"
+	description = "A Mutant toxin."
+	color = "#5EFF3B" //RGB: 94, 255, 59
+	race = /datum/species/smutant
+	mutationtext = "<span class='danger'>The pain subsides. You feel... unified.</span>"
```
Comment (out-of-phaze) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm:880]:
```suggestion
	mutationtext = span_danger("The pain subsides. You feel... unified.")
```

File: modular_citadel/code/modules/client/loadout/__donator.dm [hunk 4]
```diff
@@ -117,14 +117,10 @@
 	new /obj/item/clothing/gloves/f13/mutant/mk2(src)
 	new /obj/item/clothing/under/f13/vaultmutie(src)
 	new /obj/item/clothing/suit/armor/f13/mutant/metal(src)
+	new /obj/item/reagent_containers/pill/FEV_solution/R0CK3T69(src)
 
 /datum/gear/donator/kits/pug
-	name = "I die to mobs"
-	path = /obj/item/storage/box/large/custom_kit/pug
```
Comment (out-of-phaze) [inline modular_citadel/code/modules/client/loadout/__donator.dm:123]: You removed the kit with the cleaving saw.

File: code/datums/diseases/f13_fev.dm [hunk 5]
```diff
@@ -84,6 +84,21 @@
 				to_chat(affected_mob, "<span class='notice'>Simple, efficient, glorious...</span>")
 				var/datum/component/mood/mood = affected_mob.GetComponent(/datum/component/mood)
 				mood.setSanity(SANITY_INSANE) // You're happy, aren't you?
+// FEV - R0CK3T69
+/datum/disease/transformation/mutant/superdebug
+	agent = "FEV-R0CK3T69 toxin strain" // The unstable one.
+	desc = "A megavirus, with a protein sheath reinforced by ionized hydrogen. This variant has been mutated by radiation and will turn the affected person into something less horrifying."
+	new_form = /mob/living/carbon/human/species/smutant
```
Comment (out-of-phaze) [inline code/datums/diseases/f13_fev.dm:91]: This is redundant because it's always overwritten by `possible_forms`.

**Review by out-of-phaze — CHANGES_REQUESTED** [review]
This clearly wasn't tested, because the stage message doesn't actually work; it has the wrong typepath.

Also, keep in mind that super mutants have:
- Complete immunity to electricity (`siemens_coeff = 0`)
- Deals 25x the minimum punch damage and 3x the maximum (1 to 10 vs 25 to 30)
- Radiation immunity
- Virus immunity
- 25% extra force with all weapons over 5 force

In the future, if you haven't tested or finished a PR, create it as a draft instead.

**Comment by Pokebutt** [comment]
> This clearly wasn't tested, because the stage message doesn't actually work; it has the wrong typepath.
> 
> Also, keep in mind that super mutants have:
> 
> * Complete immunity to electricity (`siemens_coeff = 0`)
> * Deals 25x the minimum punch damage and 3x the maximum (1 to 10 vs 25 to 30)
> * Radiation immunity
> * Virus immunity
> * 25% extra force with all weapons over 5 force
> 
> In the future, if you haven't tested or finished a PR, create it as a draft instead.

I wanna clarify that shit was a debug item that I wound up just scrapping and turning into mutation toxin for streamlining

**Pull request closed without merging**